packages/perseus-editor/src/widgets/interactive-graph-editor/interactive-graph-editor.tsx

```diff
@@ -178,15 +164,11 @@ class InteractiveGraphEditor extends React.Component<Props> {
     displayName = "InteractiveGraphEditor";
     className = "perseus-widget-interactive-graph";
 
-    static defaultProps: DefaultProps = {
-        ...InteractiveGraph.defaultProps,
+    static defaultProps: InteractiveGraphDefaultWidgetOptions & {
+        valid: boolean;
+    } = {
+        ...interactiveGraphLogic.defaultWidgetOptions,
         valid: true,
-        backgroundImage: defaultBackgroundImage,
-        showTooltips: false,
-        correct: {
-            type: InteractiveGraph.defaultProps.graph.type,
```

Comment:
Rather than depending on the default props from the IG widget, I copied those values. I think that's fine? Alternatively I could update IG to pull from this file? `perseus-core` can't depend on `perseus`.

Ultimately though I think this should be fine?

Reply:
I think that's good. The default for the editor is a much different thing that the default for runtime anyways. In fact, I'd hope we _never_ have data that's published where `type` is missing... that feels like invalid data for learner-facing content.